app.json

```diff
@@ -1,7 +1,16 @@
 {
     "name": "Hello Books",
     "description": "A system that allows efficient, cost-effective management of Books for Libraries.",
+    "scripts": {
+        "postdeploy": "export APP_URL=https://$HEROKU_APP_NAME.herokuapp.com"
```

Comment:
Please where are all these scripts and environment variables coming from? and why are they added ?

Reply:
removed